### PR TITLE
adds the flaming moe a new animated bar drink

### PR DESCRIPTION
**Pull request by boodaliboo**
a drink ive wanted to make for months
https://cdn.discordapp.com/attachments/713291733303164938/713302316224086026/2020-05-22_14-08-00.mp4
moe 2 is the one in  use
it currently suppreses hallucinations and lessens stuns slightly and heats you up
and has a small chance to remove some toxins damage
made with tequila creme de menthe and synaptizine heated to 500

#### Changelog

:cl:  
rscadd: Added a new bar drink
mageadd: added animated icons for the flaming moe
/:cl:

### CODE REVIEW

**Comment by boodaliboo** [comment]
can someone rerun travis

**Comment by iWuna** [comment]
lol, nice

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 1]
```diff
@@ -2135,3 +2135,31 @@ All effects don't start immediately, but rather get worse over time; the rate is
...
+	glass_desc = "a amazing concotion of various different bar drinks and a secret ingredient"
+	
+/datum/reagent/consumable/ethanol/flaming_moe/on_mob_life(mob/living/carbon/M)
+	M.drowsyness = max(M.drowsyness-5, 0)
+	M.AdjustStun(-20, FALSE)
+	M.AdjustKnockdown(-20, FALSE)
+	M.AdjustUnconscious(-20, FALSE)
+	M.AdjustImmobilized(-20, FALSE)
+	M.AdjustParalyzed(-20, FALSE)
+	if(holder.has_reagent(/datum/reagent/toxin/mindbreaker))
```
Comment (iWuna) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm:2174]: why

**Review by iWuna — APPROVED** [review]
Checked code etc, looks fine, but i still dunno why it shold remove LCD from hooman

**Comment by alexkar598** [comment]
Not a fan of stun reduction

**Comment by MacHac** [comment]
Chug Flaming Moes before a fight and be immune to go horizontal for a few minutes?

Also, shouldn't it be the Flaming Homer?

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 2]
```diff
@@ -2135,3 +2135,31 @@ All effects don't start immediately, but rather get worse over time; the rate is
...
+/datum/reagent/consumable/ethanol/flaming_moe
+	name = "Flaming Moe"
+	description = "The drink that always keeps you coming back for Moe."
+	boozepwr = 38
+	color = "#FFF1B2"
+	quality = DRINK_FANTASTIC
+	taste_description = "tequila, creme de menthe, and a hint of medicine?"
+	glass_icon_state = "Flaming_moe2"
+	glass_name = "Flaming Moe"
+	glass_desc = "a amazing concotion of various different bar drinks and a secret ingredient"
```
Comment (Ktlwjec1) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: an
concoction

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm (hunk 1 above)
Comment (boodaliboo) [reply code/modules/reagents/chemistry/reagents/alcohol_reagents.dm:2174]: i copypasted some other code or something its like 4 am

**Comment by boodaliboo** [comment]
> Chug Flaming Moes before a fight and be immune to go horizontal for a few minutes?
> 
> Also, shouldn't it be the Flaming Homer?

it technically should be but flaming moe sounds better

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 3]
```diff
@@ -2153,3 +2153,28 @@ All effects don't start immediately, but rather get worse over time; the rate is
...
+/datum/reagent/consumable/ethanol/flaming_moe
+	name = "Flaming Moe"
+	description = "The drink that always keeps you coming back for Moe."
+	boozepwr = 38
+	color = "#FFF1B2"
+	quality = DRINK_FANTASTIC
+	taste_description = "tequila, creme de menthe, and a hint of medicine?"
+	glass_icon_state = "Flaming_moe2"
+	glass_name = "Flaming Moe"
+	glass_desc = "an amazing concotion of various different bar drinks and a secret ingredient"
```
Comment (Ktlwjec1) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: concoction

**Comment by Hopekz** [comment]
Why is this PR adding like 40 icons to icons/drinks.dmi?

**Comment by iWuna** [comment]
> Why is this PR adding like 40 icons to icons/drinks.dmi?

because it's contains over 40 icon animations?

**Review by JamieD1 — CHANGES_REQUESTED** [review]
Code good, ident needs to review sprites.

**Review by nesisknesis — APPROVED** [review]

**Pull request merged**